Source/Core/DolphinQt/Config/ARCodeWidget.cpp

```diff
@@ -113,6 +114,8 @@ void ARCodeWidget::OnContextMenuRequested()
   QMenu menu;
 
   menu.addAction(tr("Sort Alphabetically"), this, &ARCodeWidget::SortAlphabetically);
+  menu.addAction(tr("Sort by enabled codes"), this, &ARCodeWidget::SortByEnabledCodes);
```

Comment:
Seems like these should be in title case or have the alphabetical option converted to sentence-case, otherwise the casing formats here are off